### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by Gelma** [comment]
@peternewman @larsoner : can you check this out, please?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16364,6 +16366,7 @@ specfy->specify
 specfying->specifying
 speciafied->specified
 specialiced->specialised, specialized,
+specialise->specialize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: British English, remove it please and add it to the issue.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16410,6 +16413,7 @@ speciy->specify
 speciyfing->specifying
 speciyfying->specifying
 speciying->specifying
+specktor->specter
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest spectre, the BE version.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -16755,7 +16755,9 @@ spacific->specific
 spacifics->specifics
 spacified->specified
 spacifies->specifies
+spageti->spaghetti
 spagetti->spaghetti
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: You may as well add `spagheti->spaghetti` while you're here.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**